apps/expo/src/app/keypadDemo/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from 'react'
+import { StyleSheet, View } from 'react-native'
+import { ScrollView, TextInput } from 'react-native-gesture-handler'
+import { SafeAreaView } from 'react-native-safe-area-context'
+
+import Keypad from '~/components/ui/keypad/Keypad'
+
+export default function KeypadDemo() {
+  const [number, setNumber] = useState('')
+  const [keypadVisible, setKeypadVisible] = useState(false)
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <View>
+        <TextInput
+          style={{ borderWidth: 1, padding: 10, marginBottom: 20, width: '100%' }}
```

Comment:
NBD, but you could move the style object to a const var up above and then reference it here and in line 25.

Reply:
This is just a demo and is only used for testing functionality of the keypad before merging. It will be deleted after PR is merged